izpack-installer/src/main/java/com/izforge/izpack/installer/gui/InstallerFrame.java

```diff
@@ -304,6 +305,9 @@ public void buildGUI()
 
         Messages messages = locales.getMessages();
         navigator.updateButtonText(messages);
+        
+        // set the frame title
+        setTitle(getTitle(installdata));
```

Comment:
You're right.
Maybe it's better to make the getTitle() override JFrame:getTitle() and remove the title parameter completely from the InstallerFrame. But I haven't found where the InstallerFrame instance is configured. Could you show me?